opnode/flags/p2p_flags.go

```diff
@@ -0,0 +1,223 @@
+package flags
+
+import (
+	"time"
+
+	"github.com/urfave/cli"
+)
+
+func p2pEnv(v string) string {
+	return prefixEnvVar("P2P_" + v)
+}
+
+var (
+	DisableP2P = cli.BoolFlag{
+		Name:     "p2p.disable",
+		Usage:    "Completely disable the P2P stack",
+		Required: false,
+		EnvVar:   p2pEnv("DISABLE"),
+	}
+	NoDiscovery = cli.BoolFlag{
+		Name:     "p2p.no-discovery",
+		Usage:    "Disable Discv5 (node discovery)",
+		Required: false,
+		EnvVar:   p2pEnv("NO_DISCOVERY"),
+	}
+	P2PPrivPath = cli.StringFlag{
+		Name: "p2p.priv.path",
+		Usage: "Read the hex-encoded 32-byte private key for the peer ID from this txt file. Created if not already exists." +
+			"Important to persist to keep the same network identity after restarting, maintaining the previous advertised identity.",
+		Required: false,
+		Value:    "opnode_p2p_priv.txt",
+		EnvVar:   p2pEnv("PRIV_PATH"),
```

Comment:
Should add `TakesFile` here? I can't actually find much documentation on what the flag does.

Reply:
It's just a hint for auto-completing file arguments in the shell (they say something about fish support, but I don't use it). Will add it.